docs/notebooks/scripts/03_example_pipeline.py

```diff
@@ -72,7 +73,7 @@
 print(f"Train score is :{pipe.score(mol_list_train,y_train):0.2F}")
 print(f"Test score is  :{pipe.score(mol_list_test, y_test):0.2F}")
 # %% [markdown]
-# Nevermind the performance, or the exact value of the prediction, this is for demonstration purposes. We can easily predict on lists of molecules
+# Nevermind the performance, or the exact value of the prediction, this is for demonstration purpures. We can easily predict on lists of molecules
```

Comment:
something to fix later on

Reply:
Ah, sorry, did we reintroduce a lot of the type fixes?

Reply:
btw, great improvement @asiomchen